{"uuid":"a9fb5dfa-5b1a-4c66-805b-b9289d16f466","historyId":"9a7df617f7add6dee819460a4d850612","fullName":"tooSmart.LoginTests.loginWithInvalidLoginAndPass","labels":[{"name":"package","value":"tooSmart.LoginTests"},{"name":"testClass","value":"tooSmart.LoginTests"},{"name":"testMethod","value":"loginWithInvalidLoginAndPass"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"automationFirst"},{"name":"subSuite","value":"tooSmart.LoginTests"},{"name":"host","value":"alexander-System-Product-Name"},{"name":"thread","value":"31169@alexander-System-Product-Name.main(1)"},{"name":"epic","value":"Проверка страницы логина"},{"name":"feature","value":"Страница логина"},{"name":"severity","value":"normal"}],"links":[],"name":"loginWithInvalidLoginAndPass","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [The values you entered are incorrect\nSign in] but found []","trace":"java.lang.AssertionError: expected [The values you entered are incorrect\nSign in] but found []\n\tat org.testng.Assert.fail(Assert.java:96)\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:137)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:453)\n\tat org.testng.Assert.assertEquals(Assert.java:463)\n\tat tooSmart.Driver.Asserts.loginErrors_aroundBody4(Asserts.java:32)\n\tat tooSmart.Driver.Asserts$AjcClosure5.run(Asserts.java:1)\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\n\tat tooSmart.Driver.Asserts.loginErrors(Asserts.java:32)\n\tat tooSmart.LoginTests.loginWithInvalidLoginAndPass(LoginTests.java:40)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)\n"},"stage":"finished","description":"Логин с невалидными Login и Password:","start":1576854219063,"stop":1576854230786,"steps":[{"name":"Заходим на страницу: http://localhost:3000/admin","status":"passed","stage":"finished","start":1576854219126,"stop":1576854219397,"parameters":[{"name":"url","value":"http://localhost:3000/admin"}]},{"name":"Попробовать залогинится: Invalid, 123456","status":"passed","stage":"finished","start":1576854219398,"stop":1576854229628,"parameters":[{"name":"login","value":"Invalid"},{"name":"pass","value":"123456"}]},{"name":"Проверка текста ошибки Login и Pass","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [The values you entered are incorrect\nSign in] but found []","trace":"java.lang.AssertionError: expected [The values you entered are incorrect\nSign in] but found []\n\tat org.testng.Assert.fail(Assert.java:96)\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:137)\n\tat org.testng.Assert.assertEquals(Assert.java:118)\n\tat org.testng.Assert.assertEquals(Assert.java:453)\n\tat org.testng.Assert.assertEquals(Assert.java:463)\n\tat tooSmart.Driver.Asserts.loginErrors_aroundBody4(Asserts.java:32)\n\tat tooSmart.Driver.Asserts$AjcClosure5.run(Asserts.java:1)\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\n\tat tooSmart.Driver.Asserts.loginErrors(Asserts.java:32)\n\tat tooSmart.LoginTests.loginWithInvalidLoginAndPass(LoginTests.java:40)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)\n"},"stage":"finished","start":1576854229707,"stop":1576854230156,"parameters":[{"name":"login","value":""},{"name":"pass","value":"The values you entered are incorrect"}]}],"attachments":[{"name":"Page screenshot","source":"8f57e840-c365-4bd5-ba3d-ce9ec1a967b9-attachment","type":"image/png"}],"parameters":[]}